controllers/kustomization_generator_test.go

```diff
@@ -57,3 +61,33 @@ func Test_secureBuildKustomization_rel_basedir(t *testing.T) {
 	_, err := secureBuildKustomization("testdata/relbase", "testdata/relbase/clusters/staging/flux-system", false)
 	g.Expect(err).ToNot(HaveOccurred())
 }
+
+func TestGeneratorWriteFile(t *testing.T) {
```

Comment:
As a follow up / extension, it would probably be good to also check if an existing Kustomize file is still overwritten with the patches from the spec of the object.

Reply:
done